assets/css/styles_feeling_responsive.scss

```diff
@@ -47,3 +47,5 @@ sitemap:
 {% endif %}
 
 @import "11_syntax-highlighting.scss";
+
+@import "12_events-calendar"
```

Comment:
Inconsistent with previous import lines by ommiting the .scss.  Missed end of line character.